packages/jest-haste-map/src/index.ts

```diff
@@ -781,8 +781,6 @@ class HasteMap extends EventEmitter {
     const Watcher: sane.Watcher =
       canUseWatchman && this._options.useWatchman
         ? WatchmanWatcher
-        : os.platform() === 'darwin'
-        ? sane.FSEventsWatcher
```

Comment:
so no fsevents, huh? do we know how that affects performance without watchman?

Reply:
As discussed on discord, it seems like it won't work with Node 12 anyway and it is currently being rewritten, so it doesn't seem like we have much of a choice.

Reply:
for anyone coming upon these comments later:

current versions won't work for node 12: https://github.com/strongloop/fsevents/pull/252
Being rewritten (to not have features we need): https://github.com/strongloop/fsevents/pull/247

Reply:
Wut? The rewrite WOULD work with node 12+; just not the current version.

Reply:
Yeah, but we want to support node 6 and 12 at the same time